BraveRewardsUI/Common/DisclaimerLinkLabel.swift

```diff
@@ -0,0 +1,134 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+/// A Disclaimer Label that allows clickable links (or data-detectors)
+final class DisclaimerLinkLabel: UITextView {
```

Comment:
This class is a both generic and specific, for example: We don't set the text in it despite it always being the same for its usages, but we specify `Link` with only choices that are available with that text, we force centre alignment, etc. So I'd say we should pick a direction for it. Either we:

- Make it fully generic, allow the `onLinkedTapped` to just give back the URL, let the controller handle it (or pass in some sort of Generic string enum which maps case to url. Let the call-site choose styling, etc. Then replace `DisclaimerView` with this.
- Make it specific to ToS & PP, giving it specific text and move it out of the Common folder

Reply:
Made it more generic.

Reply:
Made it way more generic.